src/content/en/2020/ecommerce.md

```diff
@@ -481,7 +484,7 @@ Marketeers love push notifications but as per author's experience that awareness
 
 In our analysis, we found that only 0.68% of desktop ecommerce sites and 0.69% of mobile ecommerce sites use web push notifications. When it comes to push notifications, it's important that customers find push notifications useful. Key to this is to request permission at right time in customer journey and not to bombard users with irrelevant notifications. To address the customer fatigue with push notifications, Chrome will automatically enroll sites with very low acceptance rates into [quieter notifications UI](https://blog.chromium.org/2020/05/protecting-chrome-users-from-abusive.html) (though exact threshold is not yet defined). Standard UI will be restored for the site when acceptance rates improve within the control group.
 
-PJ Mclachlan (Product Manager, Google) has talked about [aiming for at least 50% acceptance rates](https://www.youtube.com/watch?v=J_t8c9HOjBc) to be in safe territory to avoid falling into quieter notifications UI and aiming for 80% and above acceptance rate. The median notifications acceptance rates for an ecommerce website is 13.6% on mobile and 13.2% on desktop. At median level, these acceptance rates have a lot to be desired. Even at 90th percentile level, numbers don't look very good (36.9% for mobile and 36.8% for desktop). Ecommerce sites can refer at this [another talking by PJ](https://www.youtube.com/watch?v=riKmez3sHaM) for recommended patterns to make sure push acceptance rates remain healthy and they are not getting caught off guard by upcoming abusive notifications changes.
+PJ Mclachlan (Product Manager, Google) has talked about [aiming for at least 50% acceptance rates](https://www.youtube.com/watch?v=J_t8c9HOjBc) to be in safe territory to avoid falling into quieter notifications UI and aiming for 80% and above acceptance rate. The median notifications acceptance rates for an ecommerce website is 13.6% on mobile and 13.2% on desktop. At median level, these acceptance rates have a lot to be desired. Even at 90th percentile level, numbers don't look very good (36.9% for mobile and 36.8% for desktop). Ecommerce sites can refer at this [talk](https://www.youtube.com/watch?v=riKmez3sHaM) for recommended patterns to make sure push acceptance rates remain healthy and they are not getting caught off guard by upcoming abusive notifications changes.
```

Comment:
Still doesn't read right. And "talk" isn't a create link (e.g. for accessibility). What about this?:

```suggestion
PJ Mclachlan (Product Manager, Google) has talked about [aiming for at least 50% acceptance rates](https://www.youtube.com/watch?v=J_t8c9HOjBc) to be in safe territory to avoid falling into quieter notifications UI and aiming for 80% and above acceptance rate. The median notifications acceptance rates for an ecommerce website is 13.6% on mobile and 13.2% on desktop. At median level, these acceptance rates have a lot to be desired. Even at 90th percentile level, numbers don't look very good (36.9% for mobile and 36.8% for desktop). Ecommerce sites can refer to this [talk for recommended patterns to make sure push acceptance rates remain healthy](https://www.youtube.com/watch?v=riKmez3sHaM) and they are not getting caught off guard by upcoming abusive notifications changes.
```